ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V100_0__Append_only_schema.sql

```diff
@@ -0,0 +1,484 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
since no data migration is here, it would be much cleaner, more readable if we would start also the oracle append only from scratch, as in h2database. what do you think?

Reply:
kept so we can test both old and new, and have just the changes and deltas in this -- once we remove old oracle completely maybe we move everything to this but that can be a follow-on.

now this follows same style as postgres

Reply:
split out now